src/utils.ts

```diff
@@ -1155,20 +1155,11 @@ export function shuffle<T>(sequence: Iterable<T>, limit = 0): Array<T> {
 
 // TODO(NODE-4936): read concern eligibility for commands should be codified in command construction
 // @see https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#read-concern
-export function commandSupportsReadConcern(command: Document, options?: Document): boolean {
+export function commandSupportsReadConcern(command: Document): boolean {
```

Comment:
Yes, and actually good thing you commented here, I recall we discussed specifically not removing this because of how it effected users of mapReduce via runCommand, now that runCommand doesn't support RC we can remove this. I will update the highlight to capture that.